build.user_example.bat

```diff
@@ -7,8 +7,8 @@ SET "MSYSTEM=MINGW32"
 SET "MSYS2_PATH_TYPE=inherit"
 REM [Optional] Specify GIT location if it is not already set in %PATH%
 SET "MPCHC_GIT=C:\Program Files\Git"
-REM [Optional] If you plan to modify the translations, install Python 2.7 and set the variable to its path
-SET "MPCHC_PYTHON=C:\Python27"
+REM [Optional] If you plan to modify the translations, install Python 3.7 and set the variable to its path
+SET "MPCHC_PYTHON=C:\Program Files\Python37"
```

Comment:
It's better to use Python 3.9.1 (currently latest version) or vaguely use Python 3.x or later